### PR TITLE
Stress Server

### PR DESCRIPTION
**Pull request by dkancalade**
this contains all of the data and scripts I used when stress testing my server.

I removed the artillery api key from the yml and un git ignored them.  
I temporarily added the reports but later git ignored them also.

### CODE REVIEW

File: database/createListingData.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
-
 file.write(listingColumnNames);
 
-// seedDB for listings table
-for (let i = 0; i < 1e7; i++) {
-  let listingId = i + 10001;
+  // for artillery testing in postListings.csv
+for (let i = 0; i < 40000; i++) {
+    let listingId = 1e7 - 50000;
+
```
Comment (abrahamesparza) [inline database/createListingData.js:17]: This file looks great, however I don't understand this for loop intention for "artillery testing in postListings.csv"

File: specs/loadTestGet.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      name: "phase 5"
+    - duration: 10
+      arrivalRate: 750
+      name: "phase 6"
+    - duration: 10
+      arrivalRate: 1500
+      name: "phase 7"
+    - duration: 600
+      arrivalRate: 2000
+      name: "phase 8"
```
Comment (abrahamesparza) [inline specs/loadTestGet.yml:33]: line 31 has the duration of 600 which is equivalent to 10 minutes, the prior durations are double digit. Are the double digit duration in seconds, or also minute?

File: specs/loadTestUpdate.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          listing_id: "{{ listing_id }}"
+          listing_type: "{{ listing_type }}"
+          listing_category: "{{ listing_category }}"
+          night_price: "{{ night_price }}"
+          avg_review: "{{ avg_review }}"
+          num_review: "{{ num_review }}"
+          num_beds: "{{ num_beds }}"
+          listing_title: "{{ listing_title  }}"
+          is_fav: "{{ is_fav }}"
+
```
Comment (abrahamesparza) [inline specs/loadTestUpdate.yml:53]: Looks great.

**Review by abrahamesparza — COMMENTED** [review]
Overall files look great. I pointed out a few lines that I didn't understand. Aside from that, great work.

File: database/createListingData.js (hunk 1 above)
Comment (dkancalade) [reply database/createListingData.js:17]: Oh what that is for is that I had to insert a bunch of data into my database for the post or update artillery test, so I created 40000 high number listings in csv format and exported them to the postListings.csv file.    I then used the csv file in my artillery yml update file.

File: specs/loadTestGet.yml (hunk 2 above)
Comment (dkancalade) [reply specs/loadTestGet.yml:33]: All are seconds.  What happened is I didn't want to wait forever for it to ramp up, but leslie did say that we should stress our system for 10 minutes, so when I got to the final phase I just jacked up the time.  I don't know if what I did was a good idea but that is what I did.

File: specs/loadTestUpdate.yml (hunk 3 above)
Comment (dkancalade) [reply specs/loadTestUpdate.yml:53]: thanks for your feedback

**Pull request merged**